### ISSUE TITLE
Unable to find python (avatarify) in the CamTwist dropdown

### ISSUE THREAD

**Issue by harpreets** [alievk/avatarify#227]
**Describe the bug**
Unable to find python (avatarify) in the CamTwist dropdown

**To Reproduce**
Changed client_requirements.txt to opencv-python==3.4.5.20 for compatibility with my OS.
Followed all of the installation steps successfully, ran Google colab and Mac Run steps.

> In the Settings section choose Confine to Application Window and select python (avatarify) from the drop-down menu.

**Info (please complete the following information):**
 - OS Max 10.12.1 

Mac environment info: 
**conda info**
active environment : avatarify
    active env location : /opt/miniconda3/envs/avatarify
            shell level : 2
       user config file : /Users/sample/.condarc
 populated config files : 
          conda version : 4.8.2
    conda-build version : not installed
         python version : 3.7.6.final.0
       virtual packages : __osx=10.12.1
       base environment : /opt/miniconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /opt/miniconda3/pkgs
                          /Users/sareen/.conda/pkgs
       envs directories : /opt/miniconda3/envs
                          /Users/sareen/.conda/envs
               platform : osx-64
             user-agent : conda/4.8.2 requests/2.22.0 CPython/3.7.6 Darwin/16.1.0 OSX/10.12.1
                UID:GID : 501:20
             netrc file : None
           offline mode : False

**conda list**
# packages in environment at /opt/miniconda3/envs/avatarify:
#
# Name                    Version                   Build  Channel
ca-certificates           2020.1.1                      0  
certifi                   2020.4.5.2               py37_0  
chardet                   3.0.4                    pypi_0    pypi
idna                      2.9                      pypi_0    pypi
libcxx                    10.0.0                        1  
libedit                   3.1.20181209         hb402a30_0  
libffi                    3.3                  h0a44026_1  
msgpack                   1.0.0                    pypi_0    pypi
msgpack-numpy             0.4.5                    pypi_0    pypi
ncurses                   6.2                  h0a44026_1  
numpy                     1.15.0                   pypi_0    pypi
opencv-python             3.4.5.20                 pypi_0    pypi
openssl                   1.1.1g               h1de35cc_0  
pip                       20.1.1                   py37_1  
python                    3.7.7                hf48f09d_4  
pyyaml                    5.1                      pypi_0    pypi
pyzmq                     19.0.0                   pypi_0    pypi
readline                  8.0                  h1de35cc_0  
requests                  2.23.0                   pypi_0    pypi
setuptools                47.3.0                   py37_0  
sqlite                    3.31.1               h5c1f38d_1  
tk                        8.6.8                ha441bb4_0  
urllib3                   1.25.9                   pypi_0    pypi
wheel                     0.34.2                   py37_0  
xz                        5.2.5                h1de35cc_0  
zlib                      1.2.11               h1de35cc_3  

**echo $PYTHONPATH**
comes up empty

**echo $PATH**
/opt/miniconda3/envs/avatarify/bin:/opt/miniconda3/condabin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin

**Screenshots**
![image](https://user-images.githubusercontent.com/1071915/84951371-56843080-b0be-11ea-9498-bbe563e4353f.png)

**Comment by bobvo23** [comment]
Can you confirm the following steps?
1/ The colab is running, and the port open messages are generated on Colab? 
2/ The bash command on MacOS completed successfully? Can you include the screenshot of the terminal to confirm?
3/ Completely quit CamTwist, reopen, follow steps to select the right windows?

**Comment by harpreets** [comment]
Just upgraded to Catalina, and it seems it asks for allowing camera access for all applications.

1. Yes, the collab is running, the port open are generated fine. Tunnel establishing through the terminal on Mac is also fine. 
2. Screenshot for bash command run
<img width="885" alt="Screen Shot 2020-07-01 at 8 26 48 PM" src="https://user-images.githubusercontent.com/1071915/86303328-8c0a3d00-bbd9-11ea-8f6a-0c0e792d9b67.png">

3. Tried that CamTwist quit/restart already. However, now after upgrading to Catalina, I'm seeing a `terminal (python - avatarify)` option in the terminal.

Here's some screenshots from `./run_mac.sh`

<img width="885" alt="Screen Shot 2020-07-01 at 8 28 12 PM" src="https://user-images.githubusercontent.com/1071915/86303335-93314b00-bbd9-11ea-8c52-9460cb238416.png">


<img width="885" alt="Screen Shot 2020-07-01 at 8 28 17 PM" src="https://user-images.githubusercontent.com/1071915/86303344-988e9580-bbd9-11ea-9b2e-88fc9fcd349f.png">

<img width="885" alt="Screen Shot 2020-07-01 at 8 28 23 PM" src="https://user-images.githubusercontent.com/1071915/86303349-9cbab300-bbd9-11ea-87d7-9c1a32dd5384.png">

**Comment by bobvo23** [comment]
I don't think it is a Catalina related issue, the problem is observed with other os as well. You are really close to the end result...
https://github.com/alievk/avatarify/issues/193

**Comment by alievk** [comment]
You may want to check out OBS, they included virtual camera in the Mac version.
It works, I checked it.
https://cdn-fastly.obsproject.com/downloads/obs-mac-26.1.dmg

**Comment by davidwebstar34** [comment]
I had the same issue where avatarify was not appearing in the dropdown I fixed it by adding terminal and camtwist to my camera security settings on Mac.
![image](https://user-images.githubusercontent.com/11534611/130028793-f6ec1d48-292b-4cbf-b4eb-85bf0f0b893e.png)
